package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mastodon/mastodon",
   "license": "AGPL-3.0-or-later",
-  "packageManager": "yarn@4.0.2",
+  "packageManager": "yarn@4.1.0",
```

Comment:
This change does not belong to this PR, please rebase without it

Reply:
Oops, done.